src/app/clusters/network-commissioning/network-commissioning.cpp

```diff
@@ -487,3 +492,48 @@ DeviceLayer::NetworkCommissioning::NetworkIterator * NullNetworkDriver::GetNetwo
 } // namespace Clusters
 } // namespace app
 } // namespace chip
+
+// These functions are ember interfaces, they should never be implemented since all network commissioning cluster functions are
+// implemented in NetworkCommissioning::Instance.
+bool emberAfNetworkCommissioningClusterAddOrUpdateThreadNetworkCallback(
```

Comment:
Is there a plan to somehow remove those ember functions? Is there even a way to mark a command as "handled" by a custom handler etc?

Reply:
There is a plan for not generating these functions.

Reply:
Ok, thanks for the info.